examples/svd/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "v850-svd"
+version = "0.1.0"
+authors = ["Roman Valls Guimera <brainstorm@nopcode.org>"]
+edition = "2018"
+
+[dependencies]
+# serde = { version = "1.0.123", features = [ "derive" ] }
+# quick-xml = { version = "0.21.0", features = [ "serialize" ] }
+yaserde = "0.5.1"
+yaserde_derive = "0.5.1"
```

Comment:
use local link with `path`

Reply:
Why?

Reply:
because here you use the published version.
So you don't check with the local version of the library so examples will not be in-sync with the code of the library.
using
```